development-docs/DEV_GUIDE.md

```diff
@@ -47,15 +47,15 @@ To build Strimzi from source the operator and Kafka code needs to be compiled in
         make clean
         make MVN_ARGS='-DskipTests -DskipIT' all
 
-4. In order to use the newly built images, you need to update the `install/cluster-operator/050-Deployment-strimzi-cluster-operator.yml` to obtain the images from your repositories on Docker Hub rather than the official Strimzi images. That can be done using the following command:
+4. In order to use the newly built images, you need to update the `install/cluster-operator/060-Deployment-strimzi-cluster-operator.yml` to obtain the images from your repositories on Docker Hub rather than the official Strimzi images. That can be done using the following command:
```

Comment:
```suggestion
4. To use the newly built images, update the `install/cluster-operator/060-Deployment-strimzi-cluster-operator.yml` to obtain the images from your repositories on Docker Hub rather than the official Strimzi images:
```